app/src/App.tsx

```diff
@@ -18,9 +18,8 @@ function App() {
 
   const [rpcUrl] = useState("https://ghostnet.ecadinfra.com");
   const [contractAddress] = useState(
-    // "KT18izS1s5Hsv9tRaB8VjxdTU6MFpLviLCrS"
-    // config.environment.development.aliases["hello-tacos"].address
-    getAliasAddress(config, "hello-tacos")
+    // getAliasAddress(config, "hello-tacos")
+    "KT1KKSAi9jaSZEF4hkjgcMDh9T924feWEKTy"
```

Comment:
Do we want to put a TODO here stating that we will change it back to `getAliasAddress(config, "hello-tacos")` later?